packages/expo-file-system/android/src/main/java/expo/modules/filesystem/FileSystemModule.kt

```diff
@@ -778,7 +775,7 @@ open class FileSystemModule(
       }
 
       val body = createRequestBody(options, decorator, fileUri.toFile())
-      return requestBuilder.method(method, body).build()
+      return method?.let { requestBuilder.method(method, body).build() }
```

Comment:
Good catch - previously this could throw NPE when `method` was `null` for some reason.

Reply:
> Good catch - previously this could throw NPE when `method` was `null` for some reason.

thank to okhttp kotlin migration help to report this compile error 😂